exercises/pig-latin/src/example.c

```diff
@@ -0,0 +1,121 @@
+#include "pig_latin.h"
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <ctype.h>
+#include <stdbool.h>
+
+static bool is_vowel(const char a)
+{
+   return a == 'a' || a == 'e' || a == 'i' || a == 'o' || a == 'u';
+}
+
+static bool starts_with_vowel(const char a, const char b)
```

Comment:
Same suggestion for parameter names here, maybe `first_letter, second_letter` or something